packages/flutter_tools/test/general.shard/runner/runner_test.dart

```diff
@@ -536,38 +555,3 @@ class WaitingCrashReporter implements CrashReporter {
     return _future;
   }
 }
-
-/// A fake [Analytics] that will be used to test
-/// the --disable-analytics flag
-class FakeAnalytics extends Fake implements Analytics {
-
-  FakeAnalytics({bool fakeTelemetryStatusOverride = true})
-      : _fakeTelemetryStatus = fakeTelemetryStatusOverride,
-        _fakeShowMessage = fakeTelemetryStatusOverride;
-
-  // Both of the members below can be initialized with [fakeTelemetryStatusOverride]
-  // because if we pass in false for the status, that means we can also
-  // assume the message has been shown before
-  bool _fakeTelemetryStatus;
-  bool _fakeShowMessage;
-
```

Comment:
This is no longer necessary since `package:unified_analytics` ships out its own fake instance